templates/bits/entryRow.tpl

```diff
@@ -0,0 +1,19 @@
+{strip}
+<tr id="{$e->id}" data-row-id="{$e->id}">
+  <td class="col-md-2">
+    {include "bits/entry.tpl" entry=$e editLink=true}
+  </td>
+  <td class="col-md-6 box-padding">
+      {foreach $e->getMainLexemes() as $lexeme}
+        {* create an array of properties*}
+        {$props[] = "{$lexeme->modelType}{$lexeme->modelNumber}{$lexeme->restriction}"}
+        {include "bits/lexemeLink.tpl" boxed=true}
+      {/foreach}
+  </td>
+  <td class="col-md-1 text-right">
+    {', '|implode:$e->getUniqueProps('mainLexemes', ['modelType', 'modelNumber', 'restriction'])}
```

Comment:
Aici e cam multă logică pentru Smarty (care ar trebui să se ocupe de prezentare). Declarații de vectori etc. Ce-ar fi să faci o nouă funcție `Entry::getModelNumbers()` care să returneze fix `$this->getUniqueProps('mainLexemes', ['modelType', 'modelNumber', 'restriction'])`? Apoi aici poți apela `{', '|implode:$e->getModelNumbers()}`, care arată mai mult a prezentare.

Reply:
Am avut de gând ca getUniqueProps() să returneze orice ne trece prin cap în funcție de ce vrem să prezentăm.

Reply:
Păi nu e rea, poate rămîne (și cîmpul de lexeme de pe pagina de editare a intrării colectează tipurile de model; cred că bucata aceea poate fi rescrisă în termeni de `getUniqueProps()`). Ziceam doar să mai adaugi o funcție ca să muți logica asta mai complexă din Smarty în PHP.

Reply:
Am mutat-o, cu două funcții distincte pentru `main` și `variants` care apelează `getUniqueProps()`.